code/modules/shuttle/navigation_computer.dm

```diff
@@ -28,11 +31,89 @@
 		var/obj/docking_port/stationary/S = V
 		if(jumpto_ports[S.id])
 			z_lock |= S.z
+	z_locked |= SSmapping.levels_by_trait(ZTRAIT_SPACE_RUINS) //after testing maybe move this to where z_locked used because some unaccesible z lvls can be added after init
+	z_locked |= SSmapping.levels_by_trait(ZTRAIT_CENTCOM) //If someone know how to better do this please 
+	z_locked |= SSmapping.levels_by_trait(ZTRAIT_AWAY)
+	z_locked |= SSmapping.levels_by_trait(ZTRAIT_REEBE)
 
 /obj/machinery/computer/camera_advanced/shuttle_docker/Destroy()
 	. = ..()
 	GLOB.navigation_computers -= src
 
+/obj/machinery/computer/camera_advanced/shuttle_docker/multitool_act(mob/living/user, obj/item/multitool/I)
+	..()
+	if (istype(I))
```

Comment:
Early return instead. Also you should be checking if the user is authorized in the first place to use the console